### PR TITLE
Beta5 release notes

### PR DESCRIPTION
**Pull request by neilweinstock**
Here are the (lengthy) release notes for beta 5.

### CODE REVIEW

File: ReleaseNotes.md [hunk 1]
```diff
@@ -1,6 +1,61 @@
...
+* Added and/or improved hover texts on a bunch of different buttons and controls
+* A bunch of updated component icons
+* Added icons to most buttons
+* Improvements to Undo behavior when adding new components (fixes #1513)
+* Improvements to scale dialog (fixes #411)
+* Removed "show all compatible" option in preset chooser (fixes #1405)
+* Cleaned up several of the example rockets
+* Added warning to launch preferences that changes only affect new sims (fixes #1497)
+* More Russian translation updates
+
```
Comment (SiboVG) [inline ReleaseNotes.md:58]: Put something about PR #1604, which is that when the auto (packed) diameter checkbox is checked, and the parent diameter changes, that the (packed) length of a mass component dynamically updates in order to preserve the (packed) volume from before the diameter change. Is useful e.g. when you have determined a parachute's volume, which should in theory remain constant even if you change the parent's diameter; you would just pack it differently.

File: ReleaseNotes.md [hunk 2]
```diff
@@ -1,6 +1,61 @@
...
+* Ensure simulations finish when running from scripts (fixes #1575)
+* Fixed Cd calculation for finsets (fixes #1440)
+* Move component config window back to same monitor as main app window first time it is opened.  No more lost config windows! (fixes #1470)
+* Rocket side view updates
+  * Don't recenter rocket when zooming (fixes #1464)
+  * Ensure full rocket is always visible and positioned correctly in rocket side view (fixes #1465)
+* Fixed root edge display in 3D view and fin templates (fixes #1227).  This matters when you are attaching fins to a nose cone or transition.
+* Better register of double-clicks in 3D view (fixes #1054)
+* Increased component analysis drag precision to three decimal digits (fixes #1476)
+* Don't include inactive stages in calculations (fixes #1477 and #1460)
```
[the remark this answers is not in the document]
Comment (neilweinstock) [reply ReleaseNotes.md:40]: Is that a new feature or a bug fix?

[the remark this answers is not in the document]
Comment (SiboVG) [reply ReleaseNotes.md:40]: I view it more as a new feature.

**Comment by JoePfeiffer** [comment]
Dang Sibo's been busy!  I can only say "thank you" yet again.

The remark on C_D seems fine.  Craig and I have been working on the documentation for the use of overrides.

**Comment by SiboVG** [comment]
> Dang Sibo's been busy! I can only say "thank you" yet again.

🙂

**Comment by neilweinstock** [comment]
All comments incorporated in latest commits.  @JoePfeiffer or @hcraigmiller, is there a wiki link for the explanation of Cd overrides I can insert?

**Comment by hcraigmiller** [comment]
Not quite yet, @JoePfeiffer dis a draft that I revised and now it's back to him for review, modification, or approval. Then, I'll create the Wiki for it. This is the latest if you want to take a look:

[Overrides and Surface Finish.Rev_02.txt](https://github.com/openrocket/openrocket/files/9543563/Overrides.and.Surface.Finish.Rev_02.txt)

**Comment by neilweinstock** [comment]
OK, let me know as soon as you have a URL; the page itself doesn't need to be finished.

**Comment by hcraigmiller** [comment]
Here's the [Wiki Override/Surface link](http://wiki.openrocket.info/Overrides_and_Surface_Finish)

**Comment by hcraigmiller** [comment]
UI Improvements, Bullet 9: "widdths" not width.
Notable Bug Fixes, Bullet 4 and Bullet 14: "finsets" not fin sets.

**Comment by hcraigmiller** [comment]
Looks good to me.

**Comment by SiboVG** [comment]
In addition to my previous comments, I would also add "Numerous bug fixes in component scaling (fixes #1649, #1651, #1653, #1661, #1662, and #1663)" in the 'Other' category. I'm still working on #1662, but will push a fix for that soon.

**Pull request merged**